docs/platform/release-notes/tizen-5-5-m3.md

```diff
@@ -0,0 +1,93 @@
+﻿# Tizen 5.5 Public M3 Release Notes
+
+Release date: Aug. 31, 2020
+
+
+- Tizen 5.5 public M3 adopts some new features for wearable, such as AOD viewer, Sticker framework, Aurum for testing-automation, and Battery-Monitor framework.
+
+
+## Application framework
+
+### New and changed features
+
+- App-defined-loader
+  - The new API set for preparing a specific app-defined loader has been provided.
+- Added widget-service API set
+  - Functions to enable and disable by widget class have been provided.
+- API set for watch viewer
+  - The new API set for watch viewer has been provided.
+- API set for AOD viewer
+  - The new API set for AOD viewer has been provided.
+- Watchface-complication
+  - The additional API set for watchface-complication has been provided.
+
+
+## Window System
+
+### New and changed features
+
+- tzsh-quickpanel
+  - tzsh_quickpanel_scrollable_state_set/get API and enumeration have been added.
+  - tzsh_quickpanel_scrollable_set/get API has been deprecated.
+
+
+## UI framework
+
+### New and changed features
+
+- Aurum has been added.
+  - Automation framework for UI testing
+- efl-theme-tizen-wearable has been updated with new UX.
+- Sticker framework
+  - Support for API to set/get display type of sticker (emoji, wallpaper) has been added.
+  - Support for API to delete sticker using URI has been added.
+  - API set for managing recent stickers history has been added.
+  - A new API for registering whenever sticker info is changed, has been added.

```

Comment:
Can it be rephrased as?
A new API for registering any change in sticker info has been added.

Reply:
will accept